src/panels/lovelace/editor/card-editor/hui-dialog-pick-card.ts

```diff
@@ -40,7 +40,7 @@ export class HuiDialogPickCard extends hassLocalizeLitMixin(LitElement) {
           ></hui-card-picker>
         </paper-dialog-scrollable>
         <div class="paper-dialog-buttons">
-          <paper-button @click="${this._skipPick}">SKIP</paper-button>
+          <paper-button @click="${this._skipPick}">CUSTOM CARD</paper-button>
```

Comment:
hmm. I prefer "MANUAL", as it is not limited to custom cards.

Reply:
meet me half way with "MANUAL CARD"? 😄

Reply:
Ok with me.

@bramkragten @zsarnett , opinions?

Reply:
Fine by me

Reply:
Ship it

Reply:
Ooooppsss...